### PR TITLE
Added Test cases and removed unused imports for Java

### PR DESCRIPTION
**Pull request by pemmasanikrishna**
Added Test cases and removed unused imports for Java

### CODE REVIEW

File: java/src/test/java/io/metaparticle/test/MetaParticleTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import org.powermock.api.mockito.PowerMockito;
+
+import io.metaparticle.AciExecutor;
+import io.metaparticle.DockerImpl;
+import io.metaparticle.Metaparticle;
+import io.metaparticle.MetaparticleExecutor;
+import io.metaparticle.annotations.Package;
+import io.metaparticle.annotations.Runtime;
+
+public class MetaParticleTest {
```
Comment (brendandburns) [inline java/src/test/java/io/metaparticle/test/MetaParticleTest.java]: nit: lower-case 'P' in Metaparticle

**Comment by brendandburns** [comment]
Generally LGTM, one small nit.

Thanks (and sorry for the delay)

**Comment by pemmasanikrishna** [comment]
@brendandburns renamed the class.

**Comment by brendandburns** [comment]
LGTM, thanks.

**Pull request merged**